.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
```

Comment:
```
pkg_resources.ContextualVersionConflict: (setuptools 12.0.5 (/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages), Requirement.parse('setuptools>=17.0'), {'coala'})
```

Looks like setuptools version in Travis didn't match the minimum required version of setuptools for coala.